src/Compilers/Core/Portable/Operations/ITryOperation.cs

```diff
@@ -30,6 +30,10 @@ public interface ITryOperation : IOperation
         /// Finally handler of the try.
         /// </summary>
         IBlockOperation Finally { get; }
+        /// <summary>
+        /// Exit label for the try. This will always be null for C#.
+        /// </summary>
+        ILabelSymbol ExitLabel { get; }
```

Comment:
>ILabelSymbol ExitLabel { get; } [](start = 8, length = 31)

It looks like also we need to add exit label for switch (Select Case in VB).

Reply:
Added.

---
In reply to: [169865494](https://github.com/dotnet/roslyn/pull/24979#discussion_r169865494) [](ancestors = 169865494)